src/common/string_util.cpp

```diff
@@ -26,6 +26,17 @@ void StringUtil::LTrim(string &str) {
 	str.erase(str.begin(), it);
 }
 
+// Removes first and last char
+void StringUtil::RemoveFirstAndLast(string &str) {
+	str = str.substr(1, str.size() - 2);
+}
+
+void StringUtil::RemoveFirstAndLastIfMatching(string &str, char c) {
+	if (str.front() == c && str.back() == c) {
```

Comment:
Could we remove these and their tests again? They are rather specific and not necessary in this case.